media/docs/quickstart.md

```diff
@@ -11,14 +11,16 @@ CUTLASS requires:
 - CMake 3.18+
 - host compiler supporting C++17 or greater (minimum g++ 7.5.0)
 - Python 3.6+
+- For the SYCL backend, an installation of the open source `DPC++` compiler, which 
```

Comment:
Might be useful to state the required date of the nightly, or say that it is only tested with the most recent nightly.

Reply:
Mentions the date now